Mobile/Android/Templates/Release_2.8.0/2214-user_quota.md

```diff
@@ -0,0 +1,29 @@
+#### User Quota
+
+#### PR: https://github.com/owncloud/android/pull/2214
+
+
+---
+
+ 
+| Test Case | Steps | Expected Result | Result | Related Comment
+|:---------:| :---- | :-------------- | :----- | :------------: |
+|**View**||||||
+| 1 | Portrait | Open drawer and check quota | Correctly displayed |  |  |
+| 2 | Landscape | Open drawer and check quota | Correctly displayed |  |  |
+|**Default options**||||||
+| 3 | Default | Open drawer and check quota | It is displayed the quota with no limit |  |  |
```

Comment:
Here would be **with limit** instead of **no limit**, wouldn't it?

Reply:
Fixed!